internal/attestation/azure/snp/validator.go

```diff
@@ -223,15 +223,6 @@ func (v *Validator) checkIDKeyDigest(ctx context.Context, report *spb.Attestatio
 	return nil
 }
 
-// nopAttestationLogger is a no-op implementation of AttestationLogger.
-type nopAttestationLogger struct{}
-
-// Infof is a no-op.
-func (nopAttestationLogger) Info(string, ...interface{}) {}
-
-// Warnf is a no-op.
-func (nopAttestationLogger) Warn(string, ...interface{}) {}
-
```

Comment:
why removing these?

Reply:
it's not needed, there's also another noop logger in there that can be used